<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortWidget</class>
 <widget class="QWidget" name="SortWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生信息排序</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="avgradioButton">
       <property name="text">
        <string>平均成绩</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="cradioButton">
       <property name="text">
        <string>c语言</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="enradioButton">
       <property name="text">
        <string>英语</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="mathradioButton">
       <property name="text">
        <string>数学</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ageradioButton">
       <property name="text">
        <string>年龄</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="sortButton">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="returnButton">
       <property name="text">
        <string>主菜单</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
